client/src/components/RestaurantsResults/RestaurantsResults.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router";
+import { getRestaurants } from "../../utils/serverCalls";
+
+import {
+  CardStyled,
+  Img,
+  Body,
+  Title,
+  P,
+  Details,
+  Price,
+  Rating,
+  Menu,
+  Directions,
+} from "./RestaurantsResultsStyles";
+
+export interface RestaurantData {
+  name: string;
+  place_id: string;
+  geometry: {
+    location: {
+      lat: number;
+      lng: number;
+    };
+  };
+  vicinity: string;
+  formatted_phone_number: string;
+  price_level: number;
+  rating: number;
+  url: string;
+  website: string;
+  opening_hours: {
+    weekday_text: [string];
+  };
+  photos: [
+    {
+      height: number;
+      html_attributions: [string];
+      photo_reference: string;
+      width: number;
+    }
+  ];
+  special_diet_ratings?: {
+    dairy_free?: number;
+    gluten_free?: number;
+    nut_free?: number;
+    pescatarian?: number;
+    vegan?: number;
+    vegetarian?: number;
+  };
+}
+
+export function RestaurantsResult() {
+  const {
+    latitude,
+    longitude,
+    keyword,
+    max_price,
+    min_price,
+    radius,
+    open_now,
+  } = useParams();
+
+  const [restaurantsData, setRestaurantsData] = useState(
+    [] as RestaurantData[]
+  );
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        let payload = {
+          latitude,
+          longitude,
+          keyword,
+          max_price,
+          min_price,
+          radius,
+          open_now,
+        };
+
+        let data = await getRestaurants(payload);
+        setRestaurantsData(data.results);
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      <ul>
+        {restaurantsData.map((restaurant) => {
+          return (
+            <CardStyled key={restaurant.place_id}>
+              <Img className="card-img" />
+              <Body>
+                <Title className="card-title"> {restaurant.name} </Title>
+                <P className="card-body"> Tagline, description, etc.</P>
```

Comment:
You just have hardcoded a string of: `Tagline, description, etc.` here, Is this because you wanted to include things like this in here? If yes, we can't get that information from Google, maybe another API down the road, but since its not available now, then I would remove it and create a ticket in the project to add it later.